test/WebSites/ActionResultsWebSite/Controllers/ActionResultsVerificationController.cs

```diff
@@ -108,5 +119,20 @@ private DummyClass CreateDummy()
                 SampleString = "Foo"
             };
         }
+
+        private DisposableDummy CreateDisposableDummy()
+        {
+            return new DisposableDummy();
+        }
+
+        private class DisposableDummy : IDisposable
+        {
+            public static int disposeCallCount = 0;
```

Comment:
You could have "holder service". Like, create a singleton service and add a dictionary to it. Then have the dummy add itself when it's created and remove itself when disposed.